aperturedb/Images.py

```diff
@@ -139,6 +139,7 @@ class Images(Entities):
         client: The database connector, perhaps as returned by `CommonLibrary.create_connector`
     """
     db_object = "_Image"
+    query_object = "Image"
```

Comment:
This can be done in an alternate way, with Querybuilder.find_command(self.db_object, params) at every place you want to build FindCommand

Reply:
Yes, I like that, but the goal was to minimize the changes. Do you want me to go and change everything to query builder?

Reply:
Yes . The problem to do it with different mechanisms can make a bigger to maintain surface area. Query builder is how it's mostly done in utilities DW , etc. 

The idea is to keep this query construction system self contained.

Reply:
changed!